src/main/java/uk/gov/hmcts/reform/hmc/helper/hmi/HmiCaseDetailsMapper.java

```diff
@@ -26,6 +31,38 @@ public HmiCaseDetails getCaseDetails(CaseDetails caseDetails, Long hearingId) {
             .caseRegistered(caseDetails.getCaseSlaStartDate())
             .caseInterpreterRequiredFlag(caseDetails.getCaseInterpreterRequiredFlag())
             .caseRestrictedFlag(caseDetails.getCaseRestrictedFlag())
+            .caseLinks(getCaseLinksArray(Arrays.asList(caseDetails.getCaseDeepLink())))
+            .casePublishedName(caseDetails.getPublicCaseName())
+            .caseAdditionalSecurityFlag(caseDetails.getCaseAdditionalSecurityFlag())
             .build();
     }
+
+    public HmiCaseDetails getCaseDetails(CaseDetails caseDetails, Integer versionNumber, Long hearingId) {
```

Comment:
Looks like the only difference is the addition of the `versionNumber` and rest of the builder is the same?
I think we should only need one `getCaseDetails()` method so can delete the previous one? Every request will have a version number in the `case_hearing_request` table.